dataactcore/models/jobTrackerInterface.py

```diff
@@ -285,3 +285,63 @@ def getSubmissionStatus(self,submissionId,interfaces):
             else:
                 return "validation_successful"
         return "unknown"
+
+    def getSubmissionById(self, submissionId):
+        """ Return submission object that matches ID"""
+        query = self.session.query(Submission).filter(Submission.submission_id == submissionId)
+        return self.runUniqueQuery(query, "No submission with that ID", "Multiple submissions with that ID")
+
+    def getSubmissionErrorInfo(self, submissionId):
```

Comment:
I wasn't sure here, normally for a set function I expect to be passing in the value it's set to.  Here I'm getting the information from the error tables and then setting it on the submission table, so neither name really seems to fit.  Maybe populateSubmissionErrorInfo?